enterprise/advanced_tables.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Advanced Tables plugin
+title_nav: Advanced Tables
+description: Add sorting functionality to tables.
+keywords: sort tables advanced advtable
+---
+
+The initial release of Advanced Tables adds sorting functionality to tables in {{site.productname}}. Tables can be sorted by row or column values using:
+* The **Sort** options in the **Table** menu.
+* The **Sort** options in the table contextual menu.
+
+**For example:**
+
+| Sort rows based on the selected column                   | Advanced Sort Dialog                                 |
+| -------------------------------------------------------- | ---------------------------------------------------- |
+| ![Advanced Tables enhanced contextual menu for sorting rows based on the selected Column (Sort > Sort table by column ascending/descending).]({{ site.baseurl }}/images/advtable_context_menu_sort.png) | ![Advanced Tables sort dialog (Sort > Advanced Sort{{site.ellps}}).]({{ site.baseurl }}/images/advtable_dialog_sort.png) |
+
+### Try our Advanced Tables Demo
+{% include codepen.html id="advtable-plugin" %}
+
+### Getting started with {{site.productname}} Advanced Tables
+
+#### Creating an account
+
+To try out **Advanced Tables** and Tiny Cloud-delivered editor, the first step is to create a free [Tiny account](https://www.tiny.cloud/download/).
```

Comment:
This reads really awkwardly? The think it's the Tiny Cloud delivered editor part.

Also should this be using the cloud variable?

Reply:
The whole thing is awkward really... but fixed?